### PR TITLE
Fix API for total submission done

### PR DESCRIPTION
**Pull request by RishabhJain2018**
@deshraj Please review the PR.

### CODE REVIEW

File: apps/analytics/views.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def get_challenge_phase_submission_analysis(request, challenge_pk, challenge_pha
     challenge_phase = get_challenge_phase_model(challenge_phase_pk)
 
     submissions = Submission.objects.filter(
-        challenge_phase=challenge_phase, challenge_phase__challenge=challenge)
+        challenge_phase=challenge_phase, challenge_phase__challenge=challenge, created_by=request.user.pk)
```
Comment (deshraj) [inline apps/analytics/views.py]: This check has to be done at the participant team level and not at the level of `created_by`. Please change this.

Comment (deshraj) [reply apps/analytics/views.py]: Also, please add test cases for this.

**Comment by deshraj** [comment]
Please add a test case where a team A has done 1 submission and other team B has done two submissions, then if I want to get the data from the API, then it should return 1 instead of 3.

**Comment by RishabhJain2018** [comment]
@deshraj Please review and merge it.

**Pull request merged**